smileCounter.py

```diff
@@ -11,6 +11,17 @@
 # To use a video file as input
 # video = cv2.VideoCapture('filename.mp4')
 
+# Text configuration
+font = cv2.FONT_HERSHEY_SIMPLEX
+textPosition = (0,0)
+fontScale = 1
+fontColor = (0,0,0)
+thickness = 3
+lineType = 2
+
+smiles_detected = 0
+TIME_TO_START_COUNTING = 2
```

Comment:
In most cases, constants should be placed in the appropriate configuration file and then imported at the top of the file (more order and code control)

We usually keep this order:
License comment (if any)
Standard module imports
External library imports
Local imports (from other project files)
Constant definitions (if needed)
The rest of the code (functions, classes, etc.)

For such a small program, let's refer to the PEP-8 standard for constants (https://peps.python.org/pep-0008/#constants) as you did and also use uppercase letters in the names of any constants separated by an underscore " _ ".

### Please place any constant values ​​at the beginning of the file, under the imports.

Reply:
Moved constants to top of the file, under the imports